internal/web_bundle/create_webpack_config.js

```diff
@@ -135,6 +135,25 @@ module.exports = (
                 }
               ]
             },
+            {
+              test: /\\.module\\.scss$/,
+              use: [
+                ${
+                  mode === "production"
+                    ? "MiniCssExtractPlugin.loader"
+                    : '"style-loader"'
+                },
+                {
+                  loader: "css-loader",
+                  options: {
+                    importLoaders: 1,
+                    modules: true,
```

Comment:
As a future step (not blocking this PR), we should add some examples of usage especially from TypeScript. I've seen some webpack plugins that automatically generate TypeScript definitions from CSS modules, but this won't give a very good developer experience in the context of Bazel since these definitions won't be accessible by the code editor.

Reply:
👍 

When I put together a PR for #44 I will review the examples and update them.  

You can have typescript include extra type roots via `tsconfig.json` so as long as editors are properly integrated with `tsc` (I know for sure intellij is) then you can get your scss module definitions to work in your editor via:

```
#882 ❯❯❯ cat tsconfig.json  | jq .compilerOptions.typeRoots
[
  "node_modules/@types",
  "./typescript/types"
]

#883 ❯❯❯ cat typescript/types/sass.d.ts 
declare module '*.scss' {
    const content: {[className: string]: string};
    export = content;
}
```

I put all my `.d.ts` files in the same root:

```
#885 ❯❯❯ cat typescript/types/BUILD.bazel 
package(default_visibility = ["//visibility:public"])

filegroup(
    name = "types",
    srcs = glob(["*.d.ts"]),
)
```

And include them via the label `//typescript/types` in my `ts_library` `srcs` attr.

Reply:
From what I see, one referring to a general type description for all CSS modules with the type `{[className: string]: string}`. That works, but something even better would be https://github.com/Jimdo/typings-for-css-modules-loader which will only allow you to use correct class names for your specific CSS module. Happy with either solution as a first approach.